.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: android
+
+sudo: false
+
+jdk: oraclejdk8
+
+env:
+  global:
+    - GRADLE_OPTS="-Xmx4g"
+
+android:
+  components:
+    # Use the latest platform tools
+    - platform-tools
+    - tools
+    - tools
```

Comment:
It's not a bug. See issue https://github.com/travis-ci/travis-ci/issues/6040#issuecomment-219367943